content/blog/hpe-greenlake-for-private-cloud-enterprise-scaling-and-orchestrating-modern-applications-for-the-enterprise.md

```diff
@@ -0,0 +1,95 @@
+
+![Screenshot 2: K8s Cluster creation and configuring scaled worker nodes](/img/screenshot-2-k8s-cluster-creation-and-configuring-scaled-worker-nodes.png "Screenshot 2: K8s Cluster creation and configuring scaled worker nodes")
+
+In screenshot 2 , I created a cluster. I selected configurations and resources, set parameters, and containers service provisions and initialized the cluster. 
+
+**Scaling worker nodes in containers service**: With containers service, you can scale worker nodes based on workload. You just increase or decrease the number of worker nodes in a running cluster to align with resource requirements.
+
+**Autoscaler in Kubernetes clusters**
+The Autoscaler adjusts the Kubernetes cluster's size based on specific conditions. It scales up when pods can't be scheduled due to resource limitations and scales down if nodes are underutilized for over 10 minutes.
+
+**Key points:**
+•	Scaling range: Defined by a minimum and maximum node count. The range is between 1 and 200 nodes. By default, autoscaling is off with equal min-max values.
+
+**Scaling criteria**: 
+
+1. Pending pods due to limited resources trigger a scale-up.
+2. Underutilized nodes for 10 minutes prompt a scale-down, but certain conditions can prevent this:
+   •	Recent scale-up within 10 minutes
+   •	Failed scale-down in the last 3 minutes
+   •	Nodes with critical pods or those facing specific constraints
+3. Autoscaler operates from the control plane, checking conditions every minute.
+
+   ![Screenshot 3: Scale cluster](/img/screenshot-3-scale-cluster.png "Screenshot 3: Scale cluster")
+
+**How to use configure:**
+
+1. Ensure the cluster is ready and familiarize yourself with autoscaler guidelines.
+2. Navigate to HPE GreenLake for Private Cloud Enterprise > Containers > Selected Cluster.
+3. Choose Scale from Actions. 
+4. Set the min-max node count values to either enable or disable autoscaling as shown in Screenshot 3.
+5. Optionally, add or remove node pools.
+6. To see the autoscaler logs, utilize the given kubectl command.
+
+**Conclusion – The perfect synergy**
+HPE GreenLake for Private Cloud Enterprise, in conjunction with Kubernetes, addresses a broad spectrum of enterprise applications. Whether dealing with brownfield applications that have evolved over time or greenfield applications that are freshly developed, this combination ensures seamless integration and deployment. containers service’s ability to scale resources up or down based on workload demands ensures that businesses can respond effectively to varying operational requirements. Additionally, the integrated framework provides a secure stack, reinforcing infrastructure integrity, governance, compliance, and application security. In essence, the union of HPE GreenLake for Private Cloud Enterprise and Kubernetes provides a comprehensive solution that caters to both existing and new enterprise applications, fostering a flexible, responsive, and secure environment.
```

Comment:
3rd sentence: Containers service's ability (with a capital C for Containers).

Also,suggest to insert a line space before the sentence "Additionally, the integrated..."